docs/content/concepts/sui-move-concepts/conventions.mdx

```diff
@@ -0,0 +1,487 @@
+---
+title: Conventions
+description: Recommended Sui Move 2024 best practices
+---
+
+The recommendations below are based on 2024 Move.
+
+## Add section titles
+
+```move
+module conventions::comments {
+  // === Imports ===
+
+  // === Friends ===
+
+  // === Errors ===
+
+  // === Constants ===
+
+  // === Structs ===
+
+  // === Public-Mutative Functions ===
+
+  // === Public-View Functions ===
+
+  // === Admin Functions ===
+
+  // === Public-Friend Functions ===
+
+  // === Private Functions ===
+
+  // === Test Functions ===
+}
+```
+
+## CRUD functions names
+
```

Comment:
Could use some content here, too. Maybe "These are the available CRUD functions:"